CONTRIBUTING.md

```diff
@@ -6,6 +6,25 @@ You found something you'd like to change? Great! Please submit a pull request an
 2. Push to the branch (`git push origin my-new-feature`)
```

Comment:
I'd move the "How to contribute" step below the general guidance about contributing. So after the "Things to remember section"

Reply:
Re-ordering done